app/assets/javascripts/views/restrooms/search.coffee

```diff
@@ -62,9 +62,13 @@ class Refuge.Restrooms.Search
 
 
   _searchCurrentLocation: =>
-    @_geocoder.getCurrentLocation().then (currentCoords) =>
-      @_updateForm(currentCoords.lat, currentCoords.long, "Current Location")
-      @_form.submit()
+    @_geocoder.getCurrentLocation()
+    .then (currentCoords) =>
```

Comment:
Is that a stylistic or syntactic choice? I believe this works properly having tested it and referring to examples such as https://stackoverflow.com/questions/27595933/coffeescript-promise-chaining-with-function-definition#answer-27595943

Thanks for reviewing!